test/containers/check-bastion

```diff
@@ -18,17 +18,17 @@
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
 
+from testvm import VirtMachine
+from testlib import *
+from verify import parent
 import os
```

Comment:
I don't like this (the whole "python files: fix header include order" commit). I really prefer builtin modules at the top, then third-party modules, and finally the project-local module imports.

Also, this order does not really make sense -- if you could already import testlib and verify here, then there's no need to fiddle with sys.path later and  importing `parent`.